File: solutions/2737-row-with-maximum-ones/solution.py
class Solution(object):
    def rowAndMaximumOnes(self, mat):
        """
        :type mat: List[List[int]]
        :rtype: List[int]
        """
        ls=[mat[i].count(1) for i in range(len(mat))]
        return [ls.index(max(ls)),max(ls)]

File: solutions/2737-row-with-maximum-ones/solution.java
class Solution {
    public int[] rowAndMaximumOnes(int[][] mat) {
        int n=0,j=0;
        for(int i=0;i<mat.length;i++)
        {
            List<Integer>ls=new ArrayList<>();
            for(int a:mat[i]) ls.add(a);
            int a=Collections.frequency(ls,1);
            if(a>n)
            {
                n=a;
                j=i;
            }
        }
        return new int[]{j,n};
    }
}
